src/scanner.cc

```diff
@@ -34,7 +34,10 @@ namespace
                 stack_size = UINT8_MAX;
             buffer[i++] = stack_size;
 
-            memcpy(&buffer[i], runback.data(), stack_size);
+            for (size_t j = 0; j < stack_size; ++j)
+            {
+                buffer[j] = runback[i + j];
```

Comment:
```suggestion
                buffer[i + j] = runback[i];
```

To match memcpy behaviour

Reply:
Tests don't agree :/